modules/admin_manual/pages/configuration/server/virus-scanner-support.adoc

```diff
@@ -2,20 +2,31 @@
 :toc: right
 :page-aliases: configuration/server/antivirus_configuration.adoc
 :clamav-url: http://www.clamav.net/index.html
+:icap-url: https://tools.ietf.org/html/rfc3507
+:c-icap-url: https://sourceforge.net/p/c-icap/wiki/ModulesConfiguration/
+:docker-url: https://hub.docker.com/r/deepdiver/icap-clamav-service
+:anti-virus-app-url: https://marketplace.owncloud.com/apps/files_antivirus
+:kav-docker-url: https://box.kaspersky.com/d/c8d8577dc2494256b45e/
 
 == Introduction
 
-{clamav-url}[ClamAV] is the only _officially_ supported virus scanner available for use with ownCloud. 
+When sharing files, security is a key aspect. The ownCloud {anti-virus-app-url}[Anti-Virus] extension helps by protecting against malicious software like trojans or viruses. It forwards files that are being uploaded to the ownCloud server to a  malware scanning engine before they are written to the storage. When a file is recognized as malicious, it can be logged and prevented from being uploaded to the server to ensure that files in ownCloud are free of malware. More sophisticated rules may be specified in 'Advanced' mode. ownCloud administrators can find the configuration options in the 'Security' settings.
+
+Out of the box, the ownCloud Anti-Virus extension works with {clamav-url}[Clam AntiVirus(ClamAV)] as the officially supported virus scanner. It detects all forms of malware including trojans, viruses, and worms and scans compressed files, executables, image files, PDF, as well as many other file types. The ownCloud Anti-Virus application relies on the underlying ClamAV virus scanning engine, to which the admin points ownCloud when configuring the application. The ClamAV virus definitions need to be kept up to date in order to provide effective protection.
+
+From version 1.0.0 on, Anti-Virus offers an ICAP integration for Enterprise installations. This means that admins can integrate their favorite enterprise-grade Antivirus Scanners through the open standard {icap-url}[Internet Content Adaptation Protocol (ICAP)]. With this set up, ownCloud can delegate the scanning of uploaded files to another machine, the ICAP server. The ICAP server then checks them and either greenlights them or, if malicious code is found, treats the offending file(s) according to its settings and notifies the ownCloud server. ownCloud can then act accordingly and reject the upload. Offloading the anti-virus scans to another dedicated server can greatly improve performance and scale compared to running the ClamAV virus scanner on the same machine as ownCloud.
```

Comment:
From version 1.0.0 on...
of what?

Reply:
Of the Anti-Virus app. Made it clearer: Starting with Anti-Virus version 1.0.0, the app also offers an ICAP integration...